src/app/zap-templates/templates/app/im-cluster-command-handler.zapt

```diff
@@ -98,11 +98,49 @@ void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aC
     switch (aClusterId)
     {
     {{#all_user_clusters}}
+    {{#if (isServer side)}}
```

Comment:
Could we just use `{{#chip_server_clusters}}` here?  If not, why not?

Reply:
Yes, I just forgot we have `chip_server_clusters`, thanks.